File: app/src/main/java/me/vipa/app/activities/notification/viewmodel/NotificationViewModel.java
package me.vipa.app.activities.notification.viewmodel;

import android.app.Activity;
import android.app.Application;

import androidx.lifecycle.AndroidViewModel;

import me.vipa.app.repository.notification.NotificationRepository;
import com.vipa.app.databinding.NotificationBinding;

import me.vipa.app.repository.notification.NotificationRepository;

public class NotificationViewModel extends AndroidViewModel {
    public NotificationViewModel(Application application, Activity activity, NotificationBinding binding) {
        super(application);
        NotificationRepository.getInstance().setAdapter(activity, binding);
    }
}

File: app/src/main/java/me/vipa/app/activities/notification/viewmodel/NotificationViewModel.kt
package me.vipa.app.activities.notification.viewmodel

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import me.vipa.app.repository.notification.NotificationRepository

class NotificationViewModel(application: Application) : AndroidViewModel(application) {
    private val notificationRepository: NotificationRepository = NotificationRepository.getInstance()

}